modules/swagger-codegen/src/main/resources/typescript-angular2/api.service.mustache

```diff
@@ -254,7 +255,7 @@ export class {{classname}} {
             method: {{httpMethod}},
             headers: headers,
 {{#bodyParam}}
-            body: {{paramName}} == null ? '' : JSON.stringify({{paramName}}), // https://github.com/angular/angular/issues/10612
+            body: {{paramName}} == null ? '' : JSON.stringify(ObjectSerializer.serialize({{paramName}}, "{{dataType}}")), // https://github.com/angular/angular/issues/10612
```

Comment:
This causes the same issue `{{returnType}}` caused. => `{{{dataType}}}` to prevent html encoding